### PR TITLE
ELS backoffice container [delivers #156926353]

### PR DESCRIPTION
**Pull request by tinyels**
## Description

Sets up App to use dynamic imports to load mymove or office based on hostname.  

To view office application:
1. Set up an alias for localhost named bolocal
2. run `make server_run` and `make office_client_run`

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [ ] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.
* [ ] Any new third party client side dependencies (e.g., login.gov, google analytics, CDN libraries, etc.) have been communicated to @willowbl00.
* [ ] Request review from a member of a different team.
* [ ] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](tbd) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +148,12 @@ The above will start the webpack dev server, serving the front-end on port 3000.
 
 Dependencies are managed by yarn. To add a new dependency, use `yarn add`
 
+### Setup: Office/admin client
+
+1. add the following line to \etc\hosts\
```
Comment (ntwyman) [inline README.md]: `/etc/host/` (you're not on windows now ;-) )

File: README.md [hunk 2]
```diff
@@ -147,6 +148,12 @@ The above will start the webpack dev server, serving the front-end on port 3000.
...
+    `127.0.0.1 bolocal`
```
Comment (ntwyman) [inline README.md]: let's change this to `officelocal`

File: README.md (hunk 1 above)
Comment (jim) [reply README.md]: I think you mean `/etc/hosts` 😄

Comment (tinyels) [reply README.md]: I had just set up these aliases in my Windows vm before writing document.

**Pull request merged**